sdcm/sct_config.py

```diff
@@ -998,12 +999,20 @@ def __init__(self):
 
             self['user_prefix'] = "{}-{}".format(user_prefix, version_tag)[:35]
 
-        # 8) update target_upgrade_version automaticlly
+        # 8) update target_upgrade_version automatically
         new_scylla_repo = self.get('new_scylla_repo', None)
         if new_scylla_repo and 'target_upgrade_version' not in self:
             self['target_upgrade_version'] = get_branch_version(new_scylla_repo)
         LOGGER.info(self.dump_config())
 
+        # 9) download rpms for update_db_packages
+        update_db_packages = self.get('update_db_packages', default=None)
+        if update_db_packages:
+            if update_db_packages.startswith('s3://') or update_db_packages.startswith('gs://'):
+                self['update_db_packages'] = download_dir_from_cloud(update_db_packages)
```

Comment:
downloading of the dir should happen in test setUp not in the module that handles SCT configuration

Reply:
and need to move the  current code that searches for local dir when the cluster is already provisioned.
Need to at least verify that files are there in setup stage in order to prevent wasting resources